subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/logging/GroupedOutputFixture.java

```diff
@@ -66,14 +66,15 @@
 
 
     private final String originalOutput;
+    private final String stripedOutput;
```

Comment:
Is this supposed to be "stripped" as we are removing the ANSI control sequences and work-in-progress area?